etc/Dockerfile.cache

```diff
@@ -1,9 +1,9 @@
+ARG MAIN_TAG=ghcr.io/viamrobotics/canon
```

Comment:
Made this an arg, as it's easier to switch out to "canon-test" or similar when doing out-of-band tests.